datastore/longhorn.go

```diff
@@ -107,6 +107,23 @@ func (s *DataStore) ValidateSetting(name, value string) (err error) {
 			}
 			return fmt.Errorf("cannot modify BackupTarget since there are existing standby volumes: %v", standbyVolumeNames)
 		}
+	case types.SettingNameBackupTargetCredentialSecret:
+		secret, err := s.kubeClient.CoreV1().Secrets(s.namespace).Get(value, metav1.GetOptions{})
+		if err != nil {
+			return err
```

Comment:
return errors.Wrapf(err, "failed to list the secret %v before modifying backup target credential secret setting", value)

Reply:
Fixed it. Thank you.